test/colony-network-auction.js

```diff
@@ -313,30 +316,28 @@ contract("Colony Network Auction", accounts => {
     it("sets correct final token price", async () => {
       await tokenAuction.finalize();
       const receivedTotal = await tokenAuction.receivedTotal();
-      const endPrice = WAD.mul(receivedTotal)
-        .div(quantity)
-        .addn(1);
+      const endPrice = WAD.mul(receivedTotal).div(quantity).addn(1); // eslint-disable-line prettier/prettier
       const finalPrice = await tokenAuction.finalPrice();
-      assert.equal(endPrice.toString(), finalPrice.toString(10));
+      expect(endPrice).to.eq.BN(finalPrice);
     });
 
     it("sets the finalized property", async () => {
       await tokenAuction.finalize();
       const finalized = await tokenAuction.finalized();
-      assert.isTrue(finalized);
+      expect(finalized).to.be.true;
     });
 
     it("all CLNY sent to the auction in bids is burned", async () => {
       const balanceBefore = await clnyToken.balanceOf(tokenAuction.address);
       const supplyBefore = await clnyToken.totalSupply();
       const receivedTotal = await tokenAuction.receivedTotal();
-      assert.isFalse(receivedTotal.isZero());
+      expect(receivedTotal.isZero()).to.be.false;
```

Comment:
Better be `expect(receivedTotal).to.be.zero`

Reply:
Yup!